### PR TITLE
Add LogQL basics

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Add LogQL basics section to Loki usage guide, to explain how LogQL are structured

- add logql basics
- add logstream selector
- add filter operators

image

### CODE REVIEW

**Review by hervenicol — APPROVED** [review]
Nice!

**Review by lyind — APPROVED** [review]
LGTM but Yet Another Query Language

**Review by MarcelMue — APPROVED** [review]

File: content/docs/observability/loki-usage.md [hunk 1]
```diff
@@ -28,6 +28,42 @@ confidentiality: public
 
 ![builder / code](../lokidoc-builder-code.png)
 
+## LogQL basics
+
+### Query anatomy
+
+```goat
```
[the remark this answers is not in the document]
Comment (TheoBrigitte) [reply content/docs/observability/loki-usage.md:35]: `goat` is ascii diagram generator https://github.com/bep/goat, I learned about that yesterday :)

There are more diagram language supported by hugo https://gohugo.io/content-management/diagrams/

File: content/docs/observability/loki-usage.md [hunk 2]
```diff
@@ -28,6 +28,42 @@ confidentiality: public
...
+!~ : regex does not match
+```
+
+### Filter operators
+
+Used to filter text within the log stream
+
+```
+|= : contains
+!= : does not contains
```
Comment (hervenicol) [inline content/docs/observability/loki-usage.md:61]:
```suggestion
!= : does not contain
```

**Pull request merged**